maintenance_timesheet/tests/test_maintenance_timesheet.py

```diff
@@ -55,18 +57,18 @@ def test_check_request_done(self):
                 'user_id': self.env.ref('base.user_admin').id,
                 'date': fields.Date.today(),
                 'unit_amount': 2})]
-        self.assertTrue(hr_timesheet.BAD_TIMESHEET_REQUEST_DONE_MSG
+        self.assertTrue(BAD_TIMESHEET_REQUEST_DONE_MSG
```

Comment:
This is going to fail if the test user language is not English. I recommend you to not check this. It's enough to check for the rise of the exception.